packages/spartan/windows.buildinfo.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can you explain why you're adding a Spartan package when it was removed from DC/OS a while back? https://github.com/dcos/dcos/pull/1672

https://github.com/dcos/spartan has been replaced by https://github.com/dcos/dcos-net

Reply:
we are aware of that and we are in the process of getting dcos-net working on windows. When it does work we will delete this package and move to dcos-net.  When we ported Spartan to windows it was still in use!

Reply:
so we will not be able to use pkgpanda for building any of our products and deploy out-of-band of pkgpanda which is what we were going to do.
There really is no incremental solution for progressing with pkgpanda at all!

Reply:
? We can merge the cross-platform Pkgpanda code with some number of working or placeholder Windows packages that exercise it, and then you can iterate on WIP packages on a dev branch, and merge them incrementally as they're ready.